### ISSUE TITLE
Routify start fails after tossr1.4.0 update.

### ISSUE THREAD

**Issue by PlainBane** [sveltech/ssr#13]
Hi,
After v1.4.0 update I'm getting: `no such file or directory, scandir /node_modules/.pnpm/tossr@1.4.0/node_modules/tossr/configs'`.
Full trace: 
```
> svelte-app@1.0.0 dev /home/martin/Server/Svelte/Routify/store-front
> run-p routify rollup


> svelte-app@1.0.0 rollup /home/martin/Server/Svelte/Routify/store-front
> rollup -cw


> svelte-app@1.0.0 routify /home/martin/Server/Svelte/Routify/store-front
> routify

[Routify] Watching src/pages
[Routify] Generated routes in 1 ms
[!] Error: ENOENT: no such file or directory, scandir '/home/martin/Server/Svelte/Routify/store-front/node_modules/.pnpm/tossr@1.4.0/node_modules/tossr/configs'
Error: ENOENT: no such file or directory, scandir '/home/martin/Server/Svelte/Routify/store-front/node_modules/.pnpm/tossr@1.4.0/node_modules/tossr/configs'
    at readdirSync (node:fs:1048:3)
    at getDetectDefaults (/home/martin/Server/Svelte/Routify/store-front/node_modules/.pnpm/configent@2.1.2/node_modules/configent/configent.js:111:45)
    at buildConfig (/home/martin/Server/Svelte/Routify/store-front/node_modules/.pnpm/configent@2.1.2/node_modules/configent/configent.js:61:41)
    at configent (/home/martin/Server/Svelte/Routify/store-front/node_modules/.pnpm/configent@2.1.2/node_modules/configent/configent.js:54:12)
    at Object.module.exports [as default] (/home/martin/Server/Svelte/Routify/store-front/node_modules/.pnpm/@roxi/routify@2.12.10/node_modules/@roxi/routify/lib/utils/config.js:7:40)
    at Object.<anonymous> (/home/martin/Server/Svelte/Routify/store-front/rollup.config.js:119:50)
    at Module._compile (node:internal/modules/cjs/loader:1091:14)
    at Object.require.extensions.<computed> [as .js] (/home/martin/Server/Svelte/Routify/store-front/node_modules/.pnpm/rollup@2.40.0/node_modules/rollup/dist/shared/loadConfigFile.js:513:20)
    at Module.load (node:internal/modules/cjs/loader:971:32)
    at Function.Module._load (node:internal/modules/cjs/loader:812:14)
```

**Comment by PlainBane** [comment]
Might not be connected to v1.4.0. Version 1.3.1 does the same. Tried using npm, same.

**Comment by jakobrosenberg** [comment]
Thanks @PlainBane. I'll get on this. Should hopefully have a fix out tonight.

**Comment by jakobrosenberg** [comment]
This is hopefully fixed in `spank` `v1.5.3` and `spassr` `v2.2.1`.

**Comment by PlainBane** [comment]
@jakobrosenberg Many thanks. Tested `tossr v1.4.1` and `spassr v2.2.1`. Works for me.
